glib/src/object.rs

```diff
@@ -1636,6 +1636,51 @@ pub trait ObjectExt: ObjectType {
     where
         F: Fn(&[Value]) -> Option<Value>;
 
+    /// Connect a closure to the signal `signal_name` on this object.
+    ///
+    /// If `after` is set to `true` then the callback will be called after the default class
+    /// handler of the signal is emitted, otherwise before.
+    ///
+    /// This fails if the signal does not exist.
+    ///
+    /// Same as `connect` but takes a `Closure` instead of a `Fn`.
+    #[doc(alias = "g_signal_connect_closure")]
+    fn connect_closure<'a, N>(
```

Comment:
Similar to the changes that happened in ObjectExt lately, this needs to be renamed to `try_connect_closure` & you need to provide a panicking variant named `connect_closure`